### PR TITLE
Updates 1

### PR DESCRIPTION
**Pull request by mrdeep1**

### CODE REVIEW

File: draft-ietf-core-new-block.xml [hunk 1]
```diff
@@ -1975,6 +1980,60 @@ o  Reference: [RFCXXXX]]]></artwork>
...
+        <t>If there is likely to be the possibility of network transient
+        losses, then the use of NON should be considered.</t>
+      </section>
+    </section>
+    <section anchor="REL" title="Examples with Reliable Transports">
+      <t>The notations provided in <xref target="legend"></xref> are used in
+      the following subsections.</t>
+
+      <section title="Q-Block1 Option">
+        <t>Let's now consider the use Q-Block1 Option with reliable transports
```
Comment (boucadair) [inline draft-ietf-core-new-block.xml]:
```suggestion
        <t>Let's now consider the use of Q-Block1 Option with a reliable transport
```

Comment (boucadair) [inline draft-ietf-core-new-block.xml]:
```suggestion
        <t>Let's now consider the use of Q-Block1 Option with a reliable transport
```

File: draft-ietf-core-new-block.xml [hunk 2]
```diff
@@ -1975,6 +1980,60 @@ o  Reference: [RFCXXXX]]]></artwork>
...
+        as shown in <xref target="rel3"></xref>. There is no acknowledgment of
+        packets at the CoAP layer, just the final result.</t>
+
+        <t><figure anchor="rel3"
+            title="Example of reliable Request with Q-Block1 Option">
```
Comment (boucadair) [inline draft-ietf-core-new-block.xml]:
```suggestion
            title="Example of Reliable Request with Q-Block1 Option">
```

File: draft-ietf-core-new-block.xml [hunk 3]
```diff
@@ -1975,6 +1980,60 @@ o  Reference: [RFCXXXX]]]></artwork>
...
+         +--------->| PUT /path T:0xf1 RT=10 QB1:1/1/1024
+         +--------->| PUT /path T:0xf2 RT=10 QB1:2/1/1024
+         +--------->| PUT /path T:0xf3 RT=10 QB1:3/0/1024
+         |<---------+ 2.04
+         |          |
+]]></artwork>
+          </figure></t>
+
+        <t>If there is likely to be the possibility of network transient
+        losses, then the use of NON should be considered.</t>
```
Comment (boucadair) [inline draft-ietf-core-new-block.xml]:
```suggestion
        losses, then the use of unrelible transport with NON should be considered.</t>
```

As this is an example, I wonder whether there is a value in having this sentence

File: draft-ietf-core-new-block.xml [hunk 4]
```diff
@@ -1975,6 +1980,60 @@ o  Reference: [RFCXXXX]]]></artwork>
...
+      </section>
+
+      <section title="Q-Block2 Option">
+        <t>An example of the use of Q-Block2 Option with reliable transports
```
Comment (boucadair) [inline draft-ietf-core-new-block.xml]:
```suggestion
        <t>An example of the use of Q-Block2 Option with a reliable transport
```

File: draft-ietf-core-new-block.xml [hunk 5]
```diff
@@ -1975,6 +1980,60 @@ o  Reference: [RFCXXXX]]]></artwork>
...
+         |     [[Observe triggered]]
+         |<---------+ 2.05 T:0xf0 O:1235 ET=22 QB2:0/1/1024
+         |<---------+ 2.05 T:0xf0 O:1235 ET=22 QB2:1/1/1024
+         |<---------+ 2.05 T:0xf0 O:1235 ET=22 QB2:2/1/1024
+         |<---------+ 2.05 T:0xf0 O:1235 ET=22 QB2:3/0/1024
+         |   ...    |
+]]></artwork>
+          </figure></t>
+
         <t>If there is likely to be the possibility of network transient
```
Comment (boucadair) [inline draft-ietf-core-new-block.xml:2038]: Idem as above.

**Review by boucadair — CHANGES_REQUESTED** [review]

File: draft-ietf-core-new-block.xml (hunk 1 above)
Comment (mrdeep1) [reply draft-ietf-core-new-block.xml]: Updated

Comment (mrdeep1) [reply draft-ietf-core-new-block.xml]: Duplicated ?

File: draft-ietf-core-new-block.xml (hunk 2 above)
Comment (mrdeep1) [reply draft-ietf-core-new-block.xml]: Updated

File: draft-ietf-core-new-block.xml (hunk 3 above)
Comment (mrdeep1) [reply draft-ietf-core-new-block.xml]: Updated

File: draft-ietf-core-new-block.xml (hunk 4 above)
Comment (mrdeep1) [reply draft-ietf-core-new-block.xml]: Updated

**Comment by mrdeep1** [comment]
Updates pushed and squashed out all the merge master notifications.

**Pull request merged**